FlyingSocks/Sources/Socket+Darwin.swift

```diff
@@ -42,6 +42,7 @@ extension Socket.FileDescriptor {
 
 extension Socket {
     static let stream = Int32(SOCK_STREAM)
+    static let datagram = Int32(SOCK_DGRAM)
```

Comment:
I wonder if it makes sense to make these `public` so clients can then use `Socket.datagram` or we introduce an enum

```swift
public enum SocketType {
    case stream
    case datagram
}
``

Reply:
Yeah, `enum` is probably cleaner. I can make that change.